src/thunderbots/software/ai/primitive/move_primitive.h

```diff
@@ -31,7 +31,7 @@ class MovePrimitive : public Primitive
     /**
      * Creates a new Move Primitive from a Primitive message
      *
-     * @param primtiive_msg The message from which to create the Move Primitive
+     * @param primitive_msg The message from which to create the Move Primitive
```

Comment:
:+1: - nice job fixing this up, I'm always super happy to see other small fixes like this going in.

Reply:
:money_mouth_face: Pay :clap: off :money_with_wings: that :moneybag: technical :clap: debt